developers/ocean.py/README.md

```diff
@@ -10,13 +10,13 @@ Well, brace yourselves for some exhilarating news! Introducing ocean.py, a Pytho
 
 ### Overview
 
-ocean.py serves as a connection layer bridging the V4 smart contracts and various components such as [Provider](https://github.com/oceanprotocol/provider), [Aquarius](https://github.com/oceanprotocol/aquarius), and [Compute to Data engine](https://github.com/oceanprotocol/operator-service) within Ocean Protocol. This pythonic library brings all these elements together, facilitating seamless integration and interaction. By acting as an intermediary, ocean.py enables developers to easily leverage the functionalities offered by Ocean Protocol, making it a valuable tool in building applications and solutions that utilize decentralized data marketplaces. Its purpose is to simplify the process of connecting with smart contracts and accessing services provided by Provider, Aquarius, and Compute to Data engine, providing a convenient and efficient development experience for users.
+ocean.py serves as a connection layer bridging the smart contracts and various components such as [Provider](https://github.com/oceanprotocol/provider), [Aquarius](https://github.com/oceanprotocol/aquarius), and [Compute to Data engine](https://github.com/oceanprotocol/operator-service) within Ocean Protocol. This pythonic library brings all these elements together, facilitating seamless integration and interaction. By acting as an intermediary, ocean.py enables developers to easily leverage the functionalities offered by Ocean Protocol, making it a valuable tool in building applications and solutions that utilize decentralized data marketplaces. Its purpose is to simplify the process of connecting with smart contracts and accessing services provided by Provider, Aquarius, and Compute to Data engine, providing a convenient and efficient development experience for users.
 
 #### Architectural point of view
```

Comment:
Delete the whole section, starting with "#### Architectural" and ending with "depths of the Ocean ecosystem".

Why: it's wrong. And takes up precious real estate. I co-wrote ocean.py over the last several years: there are no "branches" in ocean.py, there's no specific affordances for data discovery or data transformation etc. It's simply a tool for data scientists. Hence, delete this section.